tests/helpers.bash

```diff
@@ -10,6 +10,15 @@ BUILD_NUM="${BUILD_NUMBER:-0}"
 JOB_BASE="${JOB_BASE_NAME:-local}"
 BUILD_ID="${JOB_BASE}-${BUILD_NUM}"
 
+# Prefer local build if binary file detected.
```

Comment:
@tgraf @scanf I pressed the wrong button when I was reviewing the changes, I meant as a comment and not something that **Request changes**. It was just a warning to @scanf that he might be using the binaries installed in `/usr/bin/` and not the local ones when running the unit tests